### DISCUSSION TITLE
what is the difference between matlab’s colamd and AMD's colamd

### DISCUSSION THREAD

**Discussion by liushang0322** [JuliaSmoothOptimizers/AMD.jl#57] [General]
it seems something different between matlab's and AMD's

**Comment by dpo** [comment]
Could you be more specific?

**Comment by liushang0322** [comment]
https://www.mathworks.com/help/releases/R2022a/matlab/ref/colamd.html
i am trying to write this example using AMD but the colamd and symamd maybe something different between matlab and AMD
